component/azstorage/azstorage.go

```diff
@@ -437,6 +437,10 @@ func (az *AzStorage) ReadFile(options internal.ReadFileOptions) (data []byte, er
 	return az.storage.ReadBuffer(options.Handle.Path, 0, 0)
 }
 
+func (az *AzStorage) ReadFileWithName(options internal.ReadFileWithNameOptions) (data []byte, err error) {
```

Comment:
API name does not sound correct, for ReadBuffer there shall already be a wrapper in azstorage.